test/devicecontroller/DefaultDeviceController.test.ts

```diff
@@ -208,6 +208,18 @@ describe('DefaultDeviceController', () => {
       // Calling it again is safe.
       await deviceController.destroy();
     });
+
+    // it('the reference to audioBufferSourceNode should be undefined when disposed', async () => {
```

Comment:
Why is this unit test commented out?

Reply:
Forgot to remove this, will push a new commit to fix this. Thanks for catching!

Reply:
Updated